docs/sections/negotiation_guide.ipynb

```diff
@@ -0,0 +1,465 @@
+    "from tac.agents.v1.examples.strategy import BaselineStrategy\n",
+    "\n",
+    "strategy = BaselineStrategy()\n",
+    "agent = BaselineAgent(name=\"tac_agent\", oef_addr=\"127.0.0.1\", oef_port=10000, strategy=strategy)\n",
+    "\n"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% \n",
+     "is_executing": false
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "\n",
+    "\n",
+    "### Registration\n",
+    "\n",
+    "This part covers the steps 1-4. That is, when the agents build their own description and register as a service to the OEF.\n",
+    "This step allows the agents to be found via search queries, and hence increasing the probability to be found by other agents. \n",
+    "\n",
+    "#### The `get_service_description(is_supply)` method\n",
+    "\n",
+    "This method generates a `Description` object of the [Python OEF SDK](https://github.com/fetchai/oef-sdk-python.git) \n",
+    "(check the documentation [here](http://oef-sdk-docs.fetch.ai/oef.html#oef.schema.Description)).\n",
+    "It is basically a data structure that holds a dictionary objects, mapping from attribute names (strings) to some values.\n",
+    "Moreover, it might refer to a [DataModel](http://oef-sdk-docs.fetch.ai/oef.html#oef.schema.DataModel) object, which defines\n",
+    "the abstract structure that a `Description` object should have. You can think of them in terms of the relational database\n",
+    "domain: a `DataModel` object corresponds to an SQL Table, whereas a `Description` object correspond to a row of that table.  \n",
+    "\n",
+    "The method is used in steps 1 and 3 by `Agent_1` and `Agent_2`, respectively.\n",
+    "\n",
+    "In the context of TAC, the `Description` for service registration looks like the following:\n"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "outputs": [],
+   "source": [
+    "from oef.schema import Description\n",
+    "\n",
+    "\n",
+    "description = Description({\n",
+    "    'tac_good_0_pbk': 1, \n",
+    "    'tac_good_1_pbk': 1, \n",
+    "    'tac_good_2_pbk': 1, \n",
+    "    'tac_good_3_pbk': 1, \n",
+    "    'tac_good_4_pbk': 1, \n",
+    "    'tac_good_5_pbk': 1, \n",
+    "    'tac_good_6_pbk': 1, \n",
+    "    'tac_good_7_pbk': 1, \n",
+    "    'tac_good_8_pbk': 1, \n",
+    "    'tac_good_9_pbk': 1\n",
+    "}, data_model=None)\n"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% \n",
+     "is_executing": false
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "\n",
+    "The argument `data_model` is set to `None`, but in the framework it is properly set depending on the context \n",
+    "That is, when we refer to a seller description, we use the `\"tac_supply\"` data model, whereas in the case of buyer description,\n",
```

Comment:
Suggest to change:
That is, when we refer to a seller description, we use the `\"tac_supply\"` data model, whereas in the case of buyer description, we use the `\"tac_demand\"` data model.
>>
That is, when we refer to a description of an agent in the seller role, we use the `\"tac_supply\"` data model (the agent supplies goods), whereas in the case of a description of an agent in the buyer role, we use the `\"tac_demand\"` data model (the agent demands goods).